services/QuillLMS/engines/evidence/app/services/evidence/gen_ai/primary_feedback/prompt_builder.rb

```diff
@@ -4,30 +4,12 @@ module Evidence
   module GenAI
     module PrimaryFeedback
       class PromptBuilder < Evidence::GenAI::PromptBuilder
-        TEMPLATE_FOLDER = 'app/services/evidence/gen_ai/primary_feedback/prompts/'
-        DEFAULT_TEMPLATE = '2024_08_19_gemini_optimized.md'
-
-        OPTIMAL_SAMPLE_COUNT = 100
-        SUBOPTIMAL_SAMPLE_COUNT = 100
-
-        private def template_variables
-          {
-            passage:,
-            plagiarism_text:,
-            stem:,
-            optimal_examples:,
-            suboptimal_examples:
-          }
+        def initialize(prompt:, history: [])
```

Comment:
Since we're not using templates and overwriting `initialize` and `run`, it might be better to inherit from `ApplicationService` directly to not pull in unused cruft.